aquarius/events/events_monitor.py

```diff
@@ -571,7 +565,7 @@ def get_and_process_event_logs_for_one_block(self, block):
             try:
                 logs = self._web3.eth.get_logs(filter_params)
                 self.process_logs(logs, block)
-            except Exception as e:
+            except Exception:
```

Comment:
should we keep the error and print it if possible?

Reply:
there's already an explicit error logged. I can print out the text if you think it's needed. I just removed the unused variable based on pre-commit issues, but let me know

Reply:
yes please, print the `e` message.